ww4ff/data/transform/base.py

```diff
@@ -42,6 +42,11 @@ def compose(*collate_modules):
     return Composition(collate_modules)
 
 
+def trim(examples: Sequence[EmplacableExample], top_db: int = 40):
+    return [ex.emplaced_audio_data(torch.from_numpy(effects.trim(ex.audio_data.cpu().numpy(),
+                                                                 top_db=top_db)[0])) for ex in examples]
```

Comment:
Can you describe this trim to me? when you say db you actually meant decibels?

Reply:
Yeah, decibels with the maximum taken as reference.